hell/entries/33-make-me-one-input-with-everything.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "#33 make me one (input) with everything"
+date: 2024-08-19
+permalink: /{{ title | slug }}/index.html
+layout: layouts/entry.njk
+author: moritzgiessmann
+badcode: '<label for="textinput">First name</label>
+	<input type="text" id="textinput" aria-label="First name" placeholder="First name" title="First name">'
+goodcode: '<label for="textinput">First name</label>
+	<input type="text" id="textinput">'
+---
+
+You might have heard the joke about the Dalai Lama who walks into a pizza shop saying “Make me one with everything”. Though the good intentions were there, in the HTML and Accessibility world, less is sometimes more.
+
+<div class="section bad">
+
+## Bad code
+
+```html
+{{ badcode | pretty }}
+```
+</div>
+
+<div class="section" id="issues">
+
+## Issues and how to fix them
+
+
+1. The `aria-label`, `placeholder`, and `title` attributes all provide similar or identical information ("First name"), leading to redundancy and potential confusion.
```

Comment:
They provide the *same* information but they have a different affect on the element. Redundancy is a problem, true, but what kind of confusion are you referring to?

Reply:
Will be clarified.